### ISSUE TITLE
Pagination is not working in Beneficiaries list and add button is also not fixed.

### ISSUE THREAD

**Issue by ankit01ojha** [openMF/web-self-service-app#118]
Pagination is not working properly in beneficiaries, it is showing all the data in one page only. And the add button is not fixed also, it is moving with scrolling in the page.

![screenshot from 2018-07-12 00-26-10](https://user-images.githubusercontent.com/21126132/42594125-e0196494-856b-11e8-93ef-a3630bacc101.png)

**Comment by Rupeshiya** [comment]
@ankit01ojha  I would like to fix this ,Can I go ahead ?

**Comment by ankit01ojha** [comment]
@Rupeshiya  yeah sure :+1:

**Comment by ankit01ojha** [comment]
@Rupeshiya, I am adding this issue as GCI task.

**Comment by Rupeshiya** [comment]
@ankit01ojha  I have tried and fixed button postion but was unable to fix pagination till now , but no worry ,will try to fix other issue  👍👍

**Comment by ankit01ojha** [comment]
@Rupeshiya okay, sure.

**Comment by jonathanzhang53** [comment]
I've been looking at this issue and I have copied the formatting of pagination from `Recent Transactions` and `Accounts` and applied it to the `Beneficiaries List`, but nothing is allowing the pagination to work. 
I don't think this is an angular pagination issue, but something with the table that's off. Will look into this more later.

**Comment by m-sameer** [comment]
@ankit01ojha I have found a task on GCI portal to fix the beneficiary button scrolling but it is fixed at the bottom. I have tested on Google Chrome and Firefox. So the issue description must be updated, correct me if I'm wrong. :smile:

**Comment by jonathanzhang53** [comment]
@m-sameer you are correct. The add button is no longer buggy. I have done a few things with the pagination on my fork, feel free to take a look if you claim the task.

**Comment by elirehema** [comment]
I would like to work on this issue

**Comment by jonathanzhang53** [comment]
Go ahead.

**Comment by jonathanzhang53** [comment]
I couldn’t find a fix so I gave up. Pagination is weird here.

**Comment by elirehema** [comment]
@jonathanzhang53 checkout my [PR](https://github.com/openMF/web-self-service-app/pull/154) i have fixed it

**Comment by elirehema** [comment]
@ankit01ojha can you merge this
